### ISSUE TITLE
Force replacement when import node pools created by NAP

### ISSUE THREAD

**Issue by nguyenhoaibao** [terraform-google-modules/terraform-google-kubernetes-engine#1177]
### TL;DR

I have a GKE cluster created by `beta-public-cluster` module, then enable the NAP for that cluster using:
```
gcloud containers clusters update --enable-autoprovisioning ...
```
then the node pool get created by the NAP succesfully.

Now when I try to import that node pool to the terraform state, the module keep force replace that node pool because of the `labels`, `metadata` and `tags`, which are set by the module automatically:
```
      ~ node_config {
          ~ labels            = { # forces replacement
              + "cluster_name" = "<cluster name>"
              + "node_pool"    = "<node pool name>"
            }
          ~ metadata          = { # forces replacement
              + "cluster_name"             = "<cluster name>"
              + "node_pool"                = "<node pool name>"
                # (1 unchanged element hidden)
            }
          ~ tags              = [ # forces replacement
              + "gke-<cluster name>",
              + "gke-<node pool name>",
            ]
```

I also tried to set:
```
  node_pools_labels = {
    <node pool name> = {}
  }
```
but it also doesn't help.

### Expected behavior

I can import the node pool created by NAP successfully.

### Observed behavior

The node pool created by NAP is forced replacement.

### Terraform Configuration

```hcl
I don't think the configuration related.
```

### Terraform Version

```sh
v1.1.5
```
